resource_customizations/apps/Deployment/actions/discovery.lua

```diff
@@ -7,4 +7,5 @@ if obj.spec.paused ~= nil then
     actions["pause"] = {paused}
 end
 actions["resume"] = {["disabled"] = not(paused)}
+actions["scale"] = {["defaultValue"] = tostring(obj.spec.replicas), ["hasParameters"] = true, ["errorMessage"] = "Enter any valid number", ["regexp"] = "^[0-9]*$", }
```

Comment:
Can this feature also extended to Argo Rollouts resources which is based on K8s Deployment? Can this be done in this PR? 

Probably worth documenting this feature and set expectations when using HPA or application auto-sync is enabled.

Reply:
This feature in argo rollouts will be done in a separate PR